modules/k3s/go.mod

```diff
@@ -0,0 +1,53 @@
+module github.com/testcontainers/testcontainers-go/modules/k3s
+
+go 1.19
+
+require (
+	github.com/testcontainers/testcontainers-go v0.20.1
```

Comment:
Could you run `go mod tidy` to verify the dependencies get updated? I think this file needs to be synced

Reply:
sure thing, doing it right away

Reply:
Doing `go mod tidy` did no changes for me though, just checked

Reply:
Oh sorry, got it, I was doing in the root directory, my bad :face_with_head_bandage: , updated the dependencies